spec/features/team/accept_an_invitation_and_sign_up_spec.rb

```diff
@@ -39,6 +39,7 @@
       it "confirms the user" do
         expect(invited_user.confirmed?).to eq(true)
         expect(invited_user.name).to eq("Ron Swanson")
+        expect(invited_user.organisation).to eq(user.organisation)
```

Comment:
This looks like another acceptance test where we're testing internals... but that's not new to this PR, so I'm happy for us to address it all in one go later. I need to look at how these spies are working.

Reply:
Yeah I put a little disclaimer in the description above. I assume that soon we'll have an "Organisation Name" displayed in the main view, but until then, the only way to test that this gets assigned would be to implement controller specs, which seems a bit pointless just for this one assertion. Also controller specs generally smell.

Reply:
Oh, sorry, I missed that. Or you edited after the fact? Either way, s'cool.

Down with controller specs for sure. Even the rails team doesn't like 'em.